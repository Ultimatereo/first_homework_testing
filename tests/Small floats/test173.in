3
2.45e-07 3.3e-08 9.5e-08 3.13e-07
2.49e-07 1e-07 2.64e-07 4.51e-07
7.7e-08 6.1e-08 1.76e-07 4.49e-07
