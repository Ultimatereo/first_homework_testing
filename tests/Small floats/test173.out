0.358791
-31.8116
13.4198
